src/main/java/com/jalasoft/jfc/model/IConverter.java

```diff
@@ -34,5 +34,14 @@
      * @throws ConvertException when the convert process is not completed.
      */
 
+    /**
+     * Convert files to another type file.
+     * @param param it receives file params.
+     * @return state of conversion.
+     * @throws ConvertException when there is an error.
+     * @throws CommandValueException when there is an error.
+     * @throws ZipJfcException when there is an error.
+     * @throws IOException when there is an error.
```

Comment:
* @throws CommandValueException when there is an invalid command.
     * @throws ConvertException when there is an invalid conversion.
     * @throws ZipJfcException when there is an invalid file path.
     * @throws IOException when there is an invalid input.